lib/pages/chat/chat.dart

```diff
@@ -258,6 +260,42 @@ class ChatController extends State<Chat>
   SuggestionsController<Map<String, String?>> suggestionsController =
       SuggestionsController();
 
+  ValueNotifier<CachedPresence?> cachedPresenceNotifier = ValueNotifier(null);
+
+  final StreamController<ConnectivityResult>
+      connectivityResultStreamController = StreamController.broadcast();
```

Comment:
this controller is not used nor disposed

Reply:
Pls remove it

Reply:
done